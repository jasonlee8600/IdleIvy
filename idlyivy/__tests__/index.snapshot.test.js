import Home from '../pages/index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });