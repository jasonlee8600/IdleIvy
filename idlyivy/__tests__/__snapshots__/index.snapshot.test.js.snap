// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col bg-white h-screen justify-center"
>
  <h2
    className="text-[#2222DD]"
  >
    Join Game!
  </h2>
</div>
`;
